### DISCUSSION TITLE
Why does a field's conditions have three toggle options?

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#13969] [Archive]
When you toggle a Field Coniditon, I noticed that there seem to be 3 toggle values. The buttons start with a`-` then toggle "on/off" like normal... What does the starting `-` value do?

https://user-images.githubusercontent.com/62306630/174404854-9b97d1a3-cc3d-4586-b640-acf0e2723f4c.mov

**Comment by rijkvanzanten** [comment]
Indeterminate (neither on or off)

**Reply by erondpowell** [reply]
Cool. What does "indeterminate" do in practice? versus on or off?

**Reply by rijkvanzanten** [reply]
In this particular case, it's the difference between "Do Nothing" and "Set it to ON / OFF"

**Reply by erondpowell** [reply]
Are there other places in the app where **Readonly**, **Hidden** and **Required** can be adjusted?
I see the difference logically, but in practice, isn't "Indeterminate" the same outcome as off?

**Reply by rijkvanzanten** [reply]
Nope! There's a small difference: 

When `readonly = true` on the field level, and `readonly = indeterminate` on the conditions, the end result is `true`, not `false`. The indeterminate state in conditions effectively means _do nothing_

**Reply by erondpowell** [reply]
In what case would we need `readonly` ?

**Reply by rijkvanzanten** [reply]
> In what case would we need `readonly` ?

Whenever you want to make a field readonly (?). F.e. you have a generated UUID on the item, you want that to be viewable / copyable, but not editable, so you make it readonly
